tests/receivers/oracledb/bundled_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+
+package tests
+
+import (
+	"fmt"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/signalfx/splunk-otel-collector/tests/testutils"
+)
+
+func TestOracledbDockerObserver(t *testing.T) {
+	testutils.SkipIfNotContainerTest(t)
+	if runtime.GOOS == "darwin" {
+		t.Skip("unable to share sockets between mac and d4m vm: https://github.com/docker/for-mac/issues/483#issuecomment-758836836")
+	}
+	tc := testutils.NewTestcase(t)
+	defer tc.PrintLogsOnFailure()
+	defer tc.ShutdownOTLPReceiverSink()
+
+	_, stop := tc.Containers(
+		testutils.NewContainer().WithContext(
+			path.Join(".", "testdata", "server"),
+		).WithName("oracledb").WithExposedPorts("1521:1521").WillWaitForHealth(5 * time.Minute),
```

Comment:
The default healthcheck cmd interval of 30s can slow this down. I found this to go healthy in <10s

```
diff --git a/tests/receivers/oracledb/testdata/server/Dockerfile b/tests/receivers/oracledb/testdata/server/Dockerfile
index 768181ae..9a5b1030 100644
--- a/tests/receivers/oracledb/testdata/server/Dockerfile
+++ b/tests/receivers/oracledb/testdata/server/Dockerfile
@@ -6,4 +6,4 @@ RUN mkdir -p /container-entrypoint-initdb.d
 
 COPY initOracleDB.sql /container-entrypoint-initdb.d/startup.sql
 
-HEALTHCHECK CMD "$ORACLE_BASE/healthcheck.sh"
\ No newline at end of file
+HEALTHCHECK --interval=2s CMD "$ORACLE_BASE/healthcheck.sh"

```

Reply:
will drop these times and see what the test runners think since they are sort of the "source of truth" for the flakiness or not of future test runs

Reply:
looks like they ran fine with the lower numbers 🎉